build/automation/lib/docker.mk

```diff
@@ -97,6 +97,7 @@ docker-build docker-image: ### Build Docker image - mandatory: NAME; optional: V
 	export VERSION=$$(make docker-image-get-version)
 	make -s file-replace-variables FILE=$$dir/Dockerfile.effective
 	docker build --rm \
+		--platform linux/amd64 \
```

Comment:
Has this got backwards compatibility ? 

Would it be worth extracting this value in to a variable and then set it dependent on what the users platform is. ?